iroh/tests/provide.rs

```diff
@@ -600,6 +629,66 @@ fn readme_path() -> PathBuf {
     Path::new(env!("CARGO_MANIFEST_DIR")).join("README.md")
 }
 
+/// A collection parser that assumes that collections are just links
+#[derive(Clone, Debug, Default)]
+pub struct CollectionsAreJustLinks;
+
+impl CollectionParser for CollectionsAreJustLinks {
+    fn parse<'a, R: AsyncSliceReader + 'a>(
+        &'a self,
+        _format: u64,
+        mut reader: R,
+    ) -> LocalBoxFuture<'_, anyhow::Result<(Box<dyn LinkStream>, CollectionStats)>> {
+        async move {
+            let data = reader.read_to_end().await?;
+            let collection = postcard::from_bytes::<Vec<Hash>>(&data)?;
+            let iter: Box<dyn LinkStream> = Box::new(ArrayLinkStream::new(collection.into()));
+            Ok((iter, Default::default()))
```

Comment:
oh, i see.  you solved this stats problem by just lying about it...  Maybe allow for `None` in the struct in this case instead of using `0` as "i don't know"